_maps/Yacht.dm

```diff
@@ -0,0 +1 @@
+#define FORCE_MAP "_maps/Butterfly.json"
```

Comment:
```suggestion
#define FORCE_MAP "_maps/Yacht.json"
```
I'll just leave this here.

Reply:
Huh... when I made the commit it was like that... thanks for catching it though